Formula/jamf-pro.rb

```diff
@@ -8,6 +8,8 @@ class JamfPro < Formula
   release = JSON.parse(File.open(File.expand_path("../../jamf-pro/release.json", __FILE__)).read)
   snapshot = JSON.parse(File.open(File.expand_path("../../jamf-pro/snapshot.json", __FILE__)).read)
 
+  depends_on "mysql" => :recommended
```

Comment:
This should probably be "mysql-client", to save a lot of space, download time and an extra system service.